jobs/JSRW_BIAS_CORRECTION_O3

```diff
@@ -30,7 +30,7 @@ export SCRIPTSsrw="${HOMEdir}/scripts"
 #
 export USHdir="${USHsrw}"  # should be removed later
 . ${USHsrw}/source_util_funcs.sh
-for sect in user nco platform workflow nco global cpl_aqm_parm cpl_aqm_parm task_run_post \
+for sect in user nco platform workflow nco global cpl_aqm_parm cpl_aqm_parm \
```

Comment:
I didn't notice the nco bit. I'll remove those, too. I like the one-line-per suggestion, too.

It might help our group more easily catch these as we aim to remove as many items from these lists as possible.